.env.example

```diff
@@ -31,9 +31,11 @@ REGISTER_TEMPLATE = mb-votingapp-signup
 REGISTER_EMAIL_TAG = votingapp_signup
 REGISTER_MB_SOURCE = votingapp
 
-MAILCHIMP_GROUP_ID = 55555
-MAILCHIMP_GROUP_NAME = VotingApp
 MC_OPT_IN_PATH = 555555
+MAILCHIMP_LIST_US = 555555
+MAILCHIMP_LIST_BR = 555555
+MAILCHIMP_LIST_MX = 555555
+MAILCHIMP_LIST_GLOBAL = 555555
```

Comment:
These are just example environment variables (since this app is able to be used for multiple campaigns). The actual environment variables are set in `.env` on the server. :)